docs/root/configuration/http_filters/router_filter.rst

```diff
@@ -81,6 +82,10 @@ connect-failure
     :ref:`config_http_filters_router_x-envoy-upstream-rq-timeout-ms` or via :ref:`route
     configuration <envoy_api_field_route.RouteAction.retry_policy>`.
 
+connect-termination
```

Comment:
This should be really called `connection-termination`, since this error is produced after we're already connected.

Reply:
Also, please note that this isn't exactly safe, since this error will happen after Envoy forwarded request, and we have no guarantee that the upstream didn't start processing it already.